.github/CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+# automatically requests pull request reviews for files matching the given pattern; the last match takes precendence
+
+*       @spacetelescope/stsci.stimage-maintainers
```

Comment:
It looks like GitHub doesn't like the dot in the team name, so we might have to replace the `.` with an underscore or something in the team and in this file (@mcara I gave you write access to the team and removed myself)